src/main/resources/application.conf

```diff
@@ -37,26 +37,30 @@ backend {
   backend = "local" // Either "JES" or "local", case insensitive
 
   // If backend is JES, must supply a 'jes' stanza:
-  //      jes
-  //          applicationName
-  //          project (i.e. google project name)
-  //          baseExecutionBucket (GCS path where things will be written to)
+  jes {
+    applicationName = ""
+    project = ""
+    baseExecutionBucket = ""
+  }
 }
 
 // If authenticating with Google (e.g. if backend is 'JES') you must supply a top level stanza 'google':
+google {
+  authScheme = "user"
 
-//      google
-//           authScheme  - Either "service" for service account or "user" for user-based, case insensitive
-                // If "user" you must supply:
-                //      userAuth
-                //           user
-                //           secretsFile
-                //           dataStoreDir (dir to cache oauth credentials)
-                // You will be asked to do the oauth browser dance
+  // If authScheme is "user"
+  userAuth {
+    user = ""
+    secretsFile = ""
+    dataStoreDir = ""
+  }
 
-                // If "service" you must supply:
-                //        p12File
-                //        serviceAccountId
+  // If authScheme is "service"
+  serviceAuth {
+    p12File = ""
+    serviceAccountId = ""
+  }
+}
 
 database {
   config = main.hsqldb
```

Comment:
I can bulk comment if you'd like though I personally found this a little easier to work with when I was trying to set up JES.  Being uncommented doesn't affect anything adversely.  If you think it's better commented, I'll comment it out